app/src/main/java/com/gh4a/widget/ReactionBar.java

```diff
@@ -266,7 +274,7 @@ public void show(String content) {
         }
 
         public void toggleOwnReaction(Reaction currentReaction) {
-            final long id = currentReaction != null ? currentReaction.id() : 0;
+            Long id = currentReaction != null ? currentReaction.id() : null;
```

Comment:
Not a required change, but I thought that using a nullable `Long` here (and in the `toggleReaction` method) could make it a bit clearer that the reaction ID may or may not be present.